### ISSUE TITLE
Highlights

### ISSUE THREAD

**Issue by extratone** [extratone/bilge#276]
# Highlights

```
https://github.com/extratone/bilge/wiki/Highlights
```

Annotation using the in-development Safari Extension, [**Highlights**](https://testflight.apple.com/join/OONmU0Z2), created by *MacStories*' Finn.

This is a relatively experimental thing I’m doing. For more “””useful””” annotation, see [**my Hypothesis profile**](https://hypothes.is/users/DavidBlue) and/or my brand new (also experimental) [Annotation Raindrop Collection](https://raindrop.io/davidblue/annotation-21356040).

*(May or may not keep [this Gist](https://gist.github.com/extratone/00e5f4a3db9708a37fa5b195e0ca9a49) updated in parallel.)*

***

## [Safari Extensions - Safari - Apple Developer](https://developer.apple.com/safari/extensions/)

> Legacy Safari Extensions (.safariextz files) built with Safari Extension Builder are not supported in Safari 13 on macOS Catalina, macOS Mojave, or macOS High Sierra. The Safari Extensions Gallery for legacy extensions will no longer be available in September, 2019. Users on macOS High Sierra or later can easily find extensions

## [iOS 15 Reviewed for My Family — The Psalms](https://bilge.world/ios-15-family-review)

> when something should happen or change. A great example: using [a simple Siri Shortcut](https://www.icloud.com/shortcuts/ac6a4587d5b54abc934631c214d81c0c) to completely disable Wi-Fi and Bluetooth instead of trusting the [unnecessarily complex conditions](https://support.apple.com/en-us/HT208086) of doing so with the buttons in Control Center. I have sought out these “hacks” (as other iOS powerusers have rather absurdly called them) for very selfish reasons.

> rigging the “buttons” which are gradually being obscured or eliminated entirely in the assumption that Apple’s automation knows better than us users when something should happen or change. A great example: using [a simple Siri Shortcut](https://www.icloud.com/shortcuts/ac6a4587d5b54abc934631c214d81c0c) to completely disable Wi-Fi and Bluetooth instead of trusting the [unnecessarily complex conditions](https://support.apple.com/en-us/HT208086) of doing so with the buttons in Control Center. I have sought out these “hacks” (as other iOS powerusers have rather absurdly called them) for very selfish reasons.

## [iOS and iPadOS 15: The MacStories Review - MacStories - Part 18](https://www.macstories.net/stories/ios-and-ipados-15-the-macstories-review/18/##weather)

> when you long-press a song in your Up Next queue, you’ll now find a new ‘Move to Top’ button in the context menu that lets you quickly put a song at the top of the queue.

> there’s a new Shared with You section in the Listen Now tab that aggregates Apple Music links shared with you in the [Messages](https://www.macstories.net/stories/ios-and-ipados-15-the-macstories-review/19/##messages) app and lets you jump back to the original message with one tap. This is my favorite implementation of Shared with You since my friends and I tend to share a lot of music links on a weekly basis, so I almost see this as “Reading List, but for music”, which totally works for me.

## [Flexibits | Fantastical for iOS | Help](https://flexibits.com/fantastical-ios/help/keyboard-shortcuts)

> ########## Main view:
> 
> *   ⌘+N: New event
>     
> *   ⌘+I: Notifications
>     
> *   ⌘+,: Settings
>     
> *   ⌘+T: Go to Today
>     
> *   ⌘+Shift+F: Change sidebar layout (large iPad devices only)
>     
> *   ⌘+Shift+R: Refresh all
>     
> *   ⌘+F: Search
>     
> *   ⌘+R: Tasks
>     
> *   ⌘+1: Day view
>     
> *   ⌘+2: Week view
>     
> *   ⌘+3: Month view
>     
> *   ⌘+4: Year view
>     
> *   Shift+Up: Previous week
>     
> *   Shift+Down: Next week
>     
> *   Shift+Left: Previous day
>     
> *   Shift+Right: Next day
>     
> *   Control+1: All Calendars
>     
> *   Control+2-9: Switch calendar set
>     
> 
> ########## All navigation sheets:
> 
> *   ⌘+.: Cancel
>     
> *   esc: Cancel
>     
> 
> ########## Task view:
> 
> *   ⌘+N: New task
> 
> ########## Event view:
> 
> *   ⌘+I: Edit
>     
> *   ⌘+Alt+Delete: Delete event
>     
> *   ⌘+.: Cancel
>     
> *   esc: Cancel
>     
> 
> ########## New item view:
> 
> *   ⌘+N: Edit parser input
>     
> *   ⌘+Up/Down: Navigate items in popover
>     
> *   Enter: Select item
>     
> *   ⌘+I - Show more
>     
> 
> ########## New/edit item view:
> 
> *   ⌘+. - Cancel
>     
> *   esc - Cancel
>     
> *   ⌘+E - Edit notes
>     
> *   ⌘+I - Show more
>     
> *   ⌘+K - Toggle between event/task
>     
> *   ⌘+Alt+Delete: Delete
>     
> *   ⌘+S: Save
>     
> *   ⌘+Enter: Save

## [iOS & iPadOS 15.1 Beta 2 Release Notes | Apple Developer Documentation](https://developer.apple.com/documentation/ios-ipados-release-notes/ios-ipados-15_1-release-notes)

> On iPad mini (6th generation), websites now load with the iPad mobile user agent by default to improve text legibility. Users can customize this setting by website in the page format menu or globally through Settings > Safari > Request Desktop Site > All Websites. (83184000)

## [Release Notes - Drafts User Guide](https://docs.getdrafts.com/changelog/)

> **Fix:** Add keyboard shortcuts (⌘-, option-⌘-,) back for Settings/Preferences and Editor Settings/Preferences.

## [Chat History — Real Life](https://reallifemag.com/chat-history/)

> I think of my quintessential Slack experience as the time a colleague interrupted a discussion of celebrity gossip to express her sincere, implacable fear of death.

## [Community — Real Life](https://reallifemag.com/bad-metaphors-community/)

> In nearly every Apple advertisement from 1984 to the invention of the iPod in 2001, computers and gadgets were presented as the necessary final ingredient to an individual’s creative genius; but as Apple shifted heavily into selling handheld devices meant for consuming media, they also began leaning on the idea that to own their products was to be a part of a community: iPhone ads barely mention technology at all, opting instead to show poignant, intimate moments with the device capturing or mediating the experience.

## [The Macro of Micro.blog // pimoore.ca](https://pimoore.ca/2021/09/30/the-macro-of.html)

> There has likely never been a better or easier time to start a blog than right now. We certainly are spoiled for choice

## [Plague of Metaphors — Real Life](https://reallifemag.com/plague-of-metaphors/)

> Diseases are represented as having a will of their own.

> these diseases have been seen as deserved

## [Yoink Brings Background Clipboard Monitoring to iOS and iPadOS 15 via Picture in Picture Workaround - MacStories](https://www.macstories.net/ios/yoink-brings-background-clipboard-monitoring-to-ios-and-ipados-15-via-picture-in-picture-workaround/)

> Several years ago back when [Pastebot](https://tapbots.com/pastebot/) was also available on iPhone, Tapbots attempted to let it run persistently in the background by [playing a silent audio track](https://www.macstories.net/iphone/pastebot-clipboard-savvy-music-syncing/) that would trick iOS into not suspending Pastebot when it was closed. The feature was promptly [shut down by Apple](https://www.macstories.net/iphone/pastebot-1-4-adds-clipboard-in-the-background-kinda/).

> Speaking of missing features: at the moment, Yoink doesn’t feature any keyboard shortcuts or Shortcuts actions to quickly activate clipboard monitoring.

## [Collateral Damage: Britt Brown on negative reviews - The Wire](https://www.thewire.co.uk/in-writing/collateral-damage/collateral-damage-negative-reviews)

> To make an impact in such an ocean requires luck, money or persuasive friends in high places. This, combined with our ability to listen to almost anything at anytime for free, has reshaped the writer’s role from one of assessment to advocacy.

## [macOS Monterey: The MacStories Review - MacStories](https://www.macstories.net/stories/macos-monterey-the-macstories-review/)

> the iPad has grown into [a powerful, modular computer](https://www.macstories.net/stories/modular-computer/), while the latest Macs now run on the same processor architecture as the iPad Pro and no longer work differently in places just because that’s the way it’s always been.

> Monterey’s focus is all about system apps, a topic near and dear to me. With the technical building blocks in place and a refined design out of the way, Monterey is one of the most tangible, user-facing payoffs of the past three years of transition. More than ever before, Apple is advancing system apps across all of its platforms at the same time. Finally, everything is everywhere.

## [My Obsidian Setup, Part 3: My 'Dashboard' Note for Quick Links, Todoist Tasks, Saving Ideas, and Keeping Track of Current Projects](https://club.macstories.net/posts/extension-284)

> What I need, however, is a tool that lets me see, at a glance, all the things I’m currently working on and stuff that’s on my mind, allowing me to quickly jump to those things and start processing them.

## [How to use SharePlay on iOS 15 | Engadget](https://www.engadget.com/shareplay-on-ios-15-hands-on-how-to-guide-170020600.html)

> For the most part, everyone you're SharePlaying with will need to have the relevant apps installed (i.e. TikTok, HBO Max, Showtime, Paramount, Apple TV etc). In some cases, they'll also need to have a subscription or at least an account.

> The pill on the top left of your screen will also turn purple to let remind you you're screen-sharing.

## [FAQ](https://docs.taio.app/##/faq)

> a WebDAV based file transfer method is also planned.

## [Trump's new social media platform found using Mastodon code: Our statement - Official Mastodon Blog](https://blog.joinmastodon.org/2021/10/trumps-new-social-media-platform-found-using-mastodon-code/)

> Mastodon is free software published under the AGPLv3 license, which requires any over-the-network service using it to make its source code and any modifications to it publicly accessible.

## [Terms of Service | Truth Social](https://www.truthsocial.com/terms-of-service)

> including excessive use of capital letters and spamming (continuous posting of repetitive text), that interferes with any party’s uninterrupted use and enjoyment of the Site or modifies, impairs, disrupts, alters, or interferes with the use, features, functions, operation, or maintenance of the Site.

## [MacStories Weekly: Issue 296](https://club.macstories.net/posts/macstories-weekly-issue-296)

> Shortcuts can’t do anything with your Apple Music subscription. It can only find and play songs you’ve added to your local library.

## [System Update](https://logicmag.io/kids/system-update/)

> Scholars have shown that for childhood to become what it is now—among other things, a protected site and a source of sentiment—a lot of things had to happen, and of course they did not happen for everyone.

> Some pieces investigate the persistence of history that an emphasis on the youthfulness of technologies and technologists obscures. One writer points out that, although the money in Silicon Valley is often presented as new, much of it is in fact inherited.

> if you want to get a glimpse into the torture chambers of the War on Terror, you can set the geolocation on your Tinder app to 19.9031 N, 75.0967 W: Guantánamo, Cuba.

> Notwithstanding the lucrative business of defining and branding distinct generations, we find that one feature of youth today is that there are far too many kinds of kids to generalize about.

## [macOS Monterey: The MacStories Review - MacStories - Part 5](https://www.macstories.net/stories/macos-monterey-the-macstories-review/5/##md-clip-for-mac)

> Shortcuts can also be run from the Terminal with the command `shortcuts run 'name-of-shortcut'`. There are commands to list, view, and sign shortcuts too. Signing is the most complex of the Terminal commands allowing users to specify the sharing mode, location of the shortcut file being signed, as well as an output destination.

## [Kara Swisher on Mark Zuckerberg, Facebook Papers](https://nymag.com/intelligencer/2021/10/kara-swisher-on-mark-zuckerberg-facebook-papers.html)

> I think Mark has tried to learn. I just think he’s not up to the task, so it doesn’t really matter if he’s tried to learn.

> He runs a big city that he thinks is running fine, but nobody has water or police or anything else.

> He’s not an asshole, and he’s never been an asshole. That’s not the problem here. It’s easy when they’re assholes.

> He’s willing to take people’s private information and tell you he knows what’s best. But he doesn’t know what’s best, and he never did. It’s so easy to make him evil. I don’t think he’s evil. That’s sort of an easy way out of it. There’s a really famous W. H. Auden quote. “Evil is unspectacular and always human.” Mark is very human. That’s the problem.

## [Git - git-config Documentation](https://git-scm.com/docs/git-config##Documentation/git-config.txt-coreexcludesFile)

> core.excludesFile
> 
> Specifies the pathname to the file that contains patterns to describe paths that are not meant to be tracked, in addition to `.gitignore` (per-directory) and `.git/info/exclude`. Defaults to `$XDG_CONFIG_HOME/git/ignore`. If `$XDG_CONFIG_HOME` is either not set or empty, `$HOME/.config/git/ignore` is used instead. See [gitignore\[5\]](/docs/gitignore).

## [The Verge turns 10: editor-in-chief Nilay Patel writes about the future - The Verge](https://www.theverge.com/22750900/verge-10-anniversary-nilay-patel-restrospective-future)

> You might spend entire days with headphones in your ears, awash in sound, but until the invention of the phonograph in 1877, _no one in the world had ever listened to music alone_ because listening to music required at least one other person to play an instrument for you_._

## [Fanboys | The Verge](https://www.theverge.com/2014/1/21/5307992/inside-the-mind-of-a-fanboy)

> Although fanboys can easily be lumped together as “angry nerds,” look closer and you’ll find that each one is like a snowflake. The reasons they’ve traveled to the fringe are personal, but also familiar. A phone might not seem to be something worth fighting over, but what it stands for most definitely is.

## [Eugen Rochko, Saint of Federated Social — The Psalms](https://bilge.world/eugen-rochko-interview)

> That won't happen. There's actually a good reason why they don't do that. It's simply because you could make a toot about one thing, have people favorite it and share it, link it from other places, and then suddenly, it says 'Heil Hitler,' or something.

## [About Twitter Blue](https://help.twitter.com/en/using-twitter/twitter-blue##tbpref)

> ################ Will Twitter Blue subscribers receive preference when receiving account support?
> 
> Our goals include providing the best customer support in our industry. We’re committed to treating everyone fairly and equitably. The Twitter rules won’t apply differently to subscribers, and they won’t get faster service on issues like content moderation.

> ################ Will Twitter Blue subscribers receive different support for enforcement issues like account blocks?
> 
> We are committed to making Twitter a safe place for everyone to participate in the public conversation. That’s why policy enforcement issues and reports by subscribers will continue to be handled by existing enforcement teams, under the current review process.

> ################ Is Twitter Blue ad-free?
> 
> No. Currently, ads continue to fund our ability to innovate as we grow this new part of our business. Twitter Blue does include an Ad-free Articles feature which offers subscribers a fast-loading, ad-free reading experience on the websites of publishers within the Twitter Blue Publisher Network. Read more about our [Ad-free Articles](/en/using-twitter/twitter-blue-ad-free-articles.html) feature.

> ################ Is dedicated support a premium feature of Twitter Blue?
> 
> Twitter Blue subscribers receive dedicated support for subscription-specific issues. That could look like help with managing a subscription, how to access your account or settings, and enrollment issues. Other issues could include troubleshooting special features only available to subscribers.

## [Minimal Web Browsers: Why You Need Less Stuff On Your Browser](https://tedium.co/2020/05/12/minimal-web-browser-argument/)

> Rather than simply letting tabs fill up, you’re encouraged to create “tasks” that let you divide different use cases up. As a writer, the way I’ve been using tasks has been to create a new one for every story I write and research, so that I can separate those ideas out without them getting in the way of everything else. (I can also hide the usual junk that sits in the pinned tabs in its own task.)

## [Twitter Blue Is for People Who Love Reading the News | WIRED](https://www.wired.com/story/twitter-blue-is-not-for-everyone/)

> In January, Twitter acquired the newsletter company Revue and Breaker, an audio social service, to build up new product offerings around [newsletter subscription](https://help.twitter.com/en/using-twitter/newsletter-subscribe) and [audio chat](https://blog.twitter.com/en_us/topics/product/2021/spaces-is-here).

> If one group of users is happy, it’s journalists, who—based on my own network, at least—are subscribing in droves. But as the company considers other premium features, it will do well to look beyond its role as a platform just for news.

## [Twitter - Keyboard shortcuts / Twitter](https://mobile.twitter.com/i/keyboard_shortcuts)

> Navigation
> 
> 
> --------------
> 
> Shortcut help
> 
> ?
> 
> Next Tweet
> 
> j
> 
> Previous Tweet
> 
> k
> 
> Page down
> 
> Space
> 
> Load new Tweets
> 
> .
> 
> Home
> 
> g
> 
> +
> 
> h
> 
> Explore
> 
> g
> 
> +
> 
> e
> 
> Notifications
> 
> g
> 
> +
> 
> n
> 
> Mentions
> 
> g
> 
> +
> 
> r
> 
> Profile
> 
> g
> 
> +
> 
> p
> 
> Likes
> 
> g
> 
> +
> 
> l
> 
> Lists
> 
> g
> 
> +
> 
> i
> 
> Direct Messages
> 
> g
> 
> +
> 
> m
> 
> Moment Maker
> 
> g
> 
> +
> 
> c
> 
> Settings
> 
> g
> 
> +
> 
> s
> 
> Bookmarks
> 
> g
> 
> +
> 
> b
> 
> Top Articles
> 
> g
> 
> +
> 
> a
> 
> Go to user…
> 
> g
> 
> +
> 
> u
> 
> Display settings
> 
> g
> 
> +
> 
> d
> 
> Actions
> 
> 
> -----------
> 
> New Tweet
> 
> n
> 
> Send Tweet
> 
> ⌘
> 
> +
> 
> Enter
> 
> New Direct Message
> 
> m
> 
> Search
> 
> /
> 
> Like
> 
> l
> 
> Reply
> 
> r
> 
> Retweet
> 
> t
> 
> Share Tweet
> 
> s
> 
> Bookmark
> 
> b
> 
> Mute account
> 
> u
> 
> Block account
> 
> x
> 
> Open Tweet details
> 
> Enter
> 
> Expand photo
> 
> o
> 
> Open/Close Messages dock
> 
> i
> 
> Media
> 
> 
> ---------
> 
> Pause/Play selected Video
> 
> k
> 
> Pause/Play selected Video
> 
> space
> 
> Mute selected Video
> 
> m
> 
> Go to Audio Dock
> 
> a
> 
> +
> 
> d
> 
> Play/Pause Audio Dock
> 
> a
> 
> +
> 
> space
> 
> Mute/Unmute Audio Dock
> 
> a
> 
> +
> 
> m

## [Apple should take a break from annual iOS and macOS updates - The Verge](https://www.theverge.com/22771079/apple-macos-monteray-ios-15-mac-iphone-software-operating-system-updates-space-out-features)

> To solve the issues of lackluster annual updates, Apple should do two things. First, it should continue to build out point releases with additional features throughout the year or even unbundle app updates from its annual releases entirely. Did the FaceTime team come up with a great new feature that’ll make pandemic chatting easier? Release it when it’s ready! Second, it should start to space out its major software releases, waiting an extra year or two between iOS or macOS versions to ensure that when it does release a “new” operating system, there are actually big changes to be excited about.

## [Are people lying more since the rise of social media and smartphones? » Nieman Journalism Lab](https://www.niemanlab.org/2021/11/are-people-lying-more-since-the-rise-of-social-media-and-smartphones/)

> In Hancock’s study, the most lies per social interaction occurred via the technology with all of these features: the phone. The fewest occurred on email, where people couldn’t communicate synchronously and the messages were recorded.

## [Famous People ##77: Happy birthday to a great old website - by Kaitlyn Tiffany - famous people](https://famouspeople.substack.com/p/famous-people-77-happy-birthday-to)

> Imagine dozens of boxes stacked like a mid-play Jenga board, and you can go inside each box if you climb a flight of slippery metal stairs, and inside each box is at least one person you know, but haven’t seen in 2, maybe 3 years, and you ask them, _How has it been?_ and they tell you, and you tell them, and it feels familiar but everyone has different hair than they used to.

## [One million strong - Substack](https://on.substack.com/p/one-million-strong)

> In short, our master plan is: 
> 
> 1.  Continue to improve Substack’s simple and powerful publishing tools, including expanding support for audio, video, and community-building features. 
>     
> 2.  Grow the Substack economy to help writers connect with the services they need to do their best work and build thriving publications, such as editing, design, insurance, and financing. 
>     
> 3.  Build, in partnership with writers, a discovery network that helps them cross-promote, collaborate, and connect with audiences independent of attention economy networks.

## [The Great Linux Tablet Hope | Revue](https://midrange.tedium.co/issues/the-great-linux-tablet-hope-877948)

> I have an iPad Pro, a 2018 model that I bought earlier this year because I felt that it was important to have an understanding of the iOS ecosystem even if I was using Android with my smartphone.

## [You.com wants to hit Google where it counts: Search - Protocol — The people, power and politics of tech](https://www.protocol.com/you-dot-com-benioff)

> It serves them up accordingly and also takes into account users' stated preference for apps.

## [You.com wants to remake the search engine - The Verge](https://www.theverge.com/2021/11/9/22770614/you-salesforce-startup-search-engine-privacy-information-control)

> On top of this organizational change, You.com’s big differentiating feature is that it lets people influence which sources they see. You can “upvote” and “downvote” specific categories, so when you run searches, you’ll see preferred sources first, neutral searches next, and downvoted sources last.

## [styleguide](https://davidblue.wtf/styleguide/)

> Our vocabulary and syntax are diverse (though we do not sacrifice _undue time_ to unearthing bizarre, senseless synonyms,) fluid, and occasionally difficult, but written as such for the purpose of colorful, substantial delivery in acknowledgement of the vast resources instantly available to the modern reader. In other words: _do not tell what can simply and easily be looked up_. With search engines as they are, this is a quickly verifiable/quantifiable factor with a given definition, summary, or abstract.

## [Money Breaks Things 💸 | Revue](https://midrange.tedium.co/issues/money-breaks-things-881230)

> I’m often left wondering what might have happened to the newsletter space had Substack remained a business that only grew as its audience did.

> To me, combining the nomadic nature of a career journalist with that of a run-your-own-business newsletter is not exactly a perfect match. And a lot of people took risks on this market knowing that, yes, it was a risk. But they still found themselves looking dumbfounded as they tried to make their exit.
